src/content/docs/infrastructure/prometheus-integrations/integrations-list/node-exporter-integration.mdx

```diff
@@ -0,0 +1,418 @@
+---
+title: 'Node Exporter integration'
+tags:
+  - Integrations
+  - Node Exporter 
+  - Prometheus
+  - host monitoring 
+freshnessValidatedDate: 2024-04-10 
+---
+
+Do you want to monitor hardware and kernel metrics for a Linux server? You can do this with the New Relic remote write integration and the Prometheus Node Exporter. When you combine these two programs with the Prometheus monitoring system, you can send data to New Relic where you can analyze it. 
+
+## Prerequisites [#prerequisites]
```

Comment:
I like Ally's approach of using "Before you begin" or "Review the requirements"

Reply:
We may want to migrate to "Before you begin" at some point, but I'll keep this "as is" for now.